docs/configuring.md

```diff
@@ -573,6 +573,29 @@ questions with default answers.
     copier -fd 'user_name=Manuel Calavera' copy template destination
     ```
 
+As alternative you can also pass the path to yaml file that contains your data. This
+cannot be used in the API.
+
+!!! example
+
+    Example CLI usage to take a yaml file as data:
+
+    ```yml
+    user_name: Alice Bob
+    age: 7
+    height: 1.83
+    ```
+
+    Calling this:
```

Comment:
Maybe a bit more explicit:

```suggestion
    Passing a data file

```

Also note the empty line. Prettier doesn't format Markdown markup within the admonition block because it's non-standard syntax and Prettier interprets it as an indented code block.